app/robots/iCubGenova04/headRetargetingParams.ini

```diff
@@ -1 +1,6 @@
-smoothingTime   0.5
+remote_control_boards   ("head")
+# Notice the order of the joint list is not wrong.
```

Comment:
This is not superclear to  be honest

Reply:
Rather than "not wrong", this specific order is necessary to have the methods ``forward`` and ``inverse kinematics`` correctly working as they are used right now.